lib/tasks/transcribe.js

```diff
@@ -263,7 +264,8 @@ class TaskTranscribe extends SttTask {
           throw new Error(`Invalid vendor ${this.vendor}`);
         }
     }
-
+    // save dedicated bugname for each endpoint
+    ep.transcribe_bugname = `${bugname}_${Date.now()}`;
```

Comment:
can we use a simple integer counter rather than Date.now() ?  It might be easier to debug if necessary

Reply:
fixed

Reply:
No We can't, due to issue that new transcribe verb can have the same name again to the old transcribe verb is being killed

Reply:
I'm confused though...seems like did make this change below?